pom.xml

```diff
@@ -83,6 +83,7 @@
         <hibernate.c3p0.version>5.6.10.Final</hibernate.c3p0.version>
         <infinispan.version>14.0.2.Final</infinispan.version>
         <infinispan.protostream.processor.version>4.5.0.Final</infinispan.protostream.processor.version>
+        <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
```

Comment:
@vmuzikar this is the unused property which I mentioned to you.

Reply:
Yes, this property was removed [here](https://github.com/keycloak/keycloak/commit/5e891951f5edadb5ad7a595e4e34203e175668c5#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L86).

I'll try to investigate. I think we need to update the `set-quarkus-version.sh` to reflect latest changes.